### PR TITLE
Create mitsubishicontroller.markdown

### PR DESCRIPTION
**Pull request by adgelbfish**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#15299

## Checklist:

- [X] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [X] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/mitsubishicontroller.markdown [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+layout: page
+title: "Mitsubishi HVAC Controller"
+description: "Instructions on how to integrate Mitsubishi GB-24a and similar with Home Assistant."
+date: 2017-12-10 21:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: daikin.png
```
Comment (frenck) [inline source/_components/mitsubishicontroller.markdown]: This logo is incorrect, right?

**Review by frenck — CHANGES_REQUESTED** [review]
@adgelbfish Thank you!
Looks great in general, just left a single note.
Could you please take a look?  👍

**Comment by adgelbfish** [comment]
@frenck thanks! It should be fixed now.

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Looks good @adgelbfish! Thank you for updating.

✅ Approved. Can be merged as soon as the parent PR gets merged in.

**Comment by frenck** [comment]
Closing this PR, since the parent PR has been closed as well.
Feel free to re-open when ready.

**Pull request closed without merging**